### DISCUSSION TITLE
[Feature Request] Colorize type string in `cast` calls

### DISCUSSION THREAD

**Discussion by Azureblade3808** [microsoft/pylance-release#2641] [Enhancement]
## Environment data

Language Server version: Pylance 2021.3.2
OS and version: Windows 7
Python version (& distribution if applicable, e.g. Anaconda): 3.8.5 (Anaconda)

## Sample code

```python
from typing import cast

l = cast("list[int]", [])
```

## Expected behavior

String `"list[int]"` in the `cast` call is colorized as a type.

## Current behavior

String `"list[int]"` in the `cast` call is colorized as a plain string.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
